<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsFilesSelectWidget</class>
 <widget class="QWidget" name="QgsFilesSelectWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>选择文件</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTableWidget" name="mTableWidget"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>保存路径</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="mSaveEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="mSaveBtn">
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
